cypress/integration/Landing.spec.js

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable no-undef */
+/// <reference types="cypress" />
+
+import testIds from '../../shared/testIds';
+import { getByTestId } from '../utils';
+
+const { landing } = testIds;
+
+describe('[Landing] section', () => {
+  describe('[Hero] CTAs', () => {
+    beforeEach(() => {
+      cy.visit('/');
+    });
+
+    it(`Should go to [/volunteer] when user clicks on [Volunteer]`, () => {
+      cy.get(getByTestId(landing.sections.hero.volunteer)).click();
+      cy.location('pathname').should('eq', '/volunteer');
+    });
+
+    it(`Should go to [/volunteer] when user clicks on [I want to sew masks]`, () => {
+      cy.get(getByTestId(landing.sections.hero.sewMasks)).click();
+      cy.location('pathname').should('eq', '/volunteer');
+    });
+
+    it(`Should go to [/request-supplies] when user clicks on [Get Masks]`, () => {
+      cy.get(getByTestId(landing.sections.hero.requestSupplies)).click();
+      cy.location('pathname').should('eq', '/request-supplies');
+    });
+
+    it(`Should contain reference to [https://rosiesews.freshdesk.com/support/] on [I Can't Sew But I Want To Help]`, () => {
+      cy.get(getByTestId(landing.sections.hero.help))
+        .should('have.attr', 'href')
+        .and('include', 'https://rosiesews.freshdesk.com/support/');
+    });
+  });
+
+  describe('[Get The Pattern] CTAs', () => {
+    Object.keys(landing.sections.afterSignUp.getThePattern).map(pattern =>
+      it(`Should have the pdf associated to [${pattern}]`, () => {
+        cy.get(getByTestId(landing.sections.afterSignUp.getThePattern[pattern]))
+          .should('have.attr', 'href')
+          .and('include', 'pdf');
+      })
+    );
+  });
+
+  describe('[Make Masks] CTAs', () => {
+    const mapPatternRoutes = {
+      '3-Layer Mask': '/patterns/3-layer-pattern/',
+      'Pocket Face Mask': '/patterns/pocket-pattern/',
+      'Mask Cover': '/patterns/mask-cover/',
+      'T-Shirt Face Mask': '/patterns/t-shirt-mask-pattern/',
+      'Convertible CareCap': '/patterns/convertible-carecap/',
+    };
+
+    beforeEach(() => {
+      cy.visit('/');
+    });
+    Object.keys(mapPatternRoutes).map(pattern =>
+      it(`Should be able to go to [${mapPatternRoutes[pattern]}] when the use clicks on [${pattern}]`, () => {
+        cy.get(
+          getByTestId(landing.sections.afterSignUp.makeMasks[pattern])
+        ).click();
+
+        // Adding this for testing on local; as opening /patterns/[xyz] causes the website to crash so the test fails
```

Comment:
You may need to check out the how-to docs for integrating cypress with gatsby here: https://www.gatsbyjs.org/docs/end-to-end-testing/

It may make the test run when navigating to /patterns/[xyz] work correctly.

Reply:
I checked that out and added `start-server-and-test` and `@testing-library/cypress` which is amazing! this will make things easier for us when it comes to CI.

However - I still can't seem to make the patterns page to work locally... Is this just me? Am I missing an env variable? 😭
![image](https://user-images.githubusercontent.com/22222078/80741286-ae200a00-8b19-11ea-8488-28c301517869.png)